spec/support/saml_auth_helper.rb

```diff
@@ -151,7 +155,7 @@ def saml_request
   end
 
   def key
-    @key ||= Rails.application.secrets.saml_client_private_key
+    @key ||= File.read("#{Rails.root}/config/saml_client_private_key.enc")
```

Comment:
Yep. I didn't change the naming in this PR. We can address that in another PR if it's an issue.